### PR TITLE
docs: Getting started with cloud providers

### PR DESCRIPTION
**Pull request by Anush008**
[Preview](https://deploy-preview-809--condescending-goldwasser-91acf0.netlify.app/documentation/hybrid-cloud/platform-deployment-options//)

### CODE REVIEW

**Comment by Anush008** [comment]
@davidmyriel, @bashofmann I saw there was a new page called `Deployment Platforms`, so I moved all the individual pages there.
https://github.com/qdrant/landing_page/pull/809/commits/888a056556cb86a10a50f98422a268bae4b9a09b

File: qdrant-landing/content/documentation/hybrid-cloud/platform-deployment-options.md [hunk 1]
```diff
@@ -5,34 +5,136 @@ weight: 4
...
+- [LKE API Reference](https://www.linode.com/docs/api/)
 
+![AWS](/documentation/cloud/cloud-providers/aws.jpg)
 
-## DigitalOcean
+## Amazon Web Services (AWS)
 
+[Amazon Elastic Kubernetes Service (Amazon EKS)](https://aws.amazon.com/eks/) is a managed service to run Kubernetes in the AWS cloud and on-premises data centers which can then be paired with Qdrant's hybrid cloud. With Amazon EKS, you can take advantage of all the performance, scale, reliability, and availability of AWS infrastructure, as well as integrations with AWS networking and security services.
+
+### More on Amazon Elastic Kubernetes Service
```
Comment (bashofmann) [inline qdrant-landing/content/documentation/hybrid-cloud/platform-deployment-options.md:26]: EKS does not automatically install CSI and Snapshot controller. You have to add this as an add-on

https://docs.aws.amazon.com/eks/latest/userguide/managing-ebs-csi.html
https://docs.aws.amazon.com/eks/latest/userguide/csi-snapshot-controller.html

Comment (bashofmann) [reply qdrant-landing/content/documentation/hybrid-cloud/platform-deployment-options.md:26]: You also need to create a storageclass manually with allowVolumeExpansion set to true, and a corresponding VolumeSnapshotClass

File: qdrant-landing/content/documentation/hybrid-cloud/platform-deployment-options.md [hunk 2]
```diff
@@ -5,34 +5,136 @@ weight: 4
...
+
+### More on Azure Kubernetes Service
+
+- [Getting Started with AKS](https://learn.microsoft.com/en-us/azure/architecture/reference-architectures/containers/aks-start-here)
+- [AKS Documentation](https://learn.microsoft.com/en-in/azure/aks/)
+- [Best Practices with AKS](https://learn.microsoft.com/en-in/azure/aks/best-practices)
+
+![Oracle Cloud Infrastructure](/documentation/cloud/cloud-providers/oracle.jpg)
 
 ## Oracle Cloud Infrastructure
```
Comment (bashofmann) [inline qdrant-landing/content/documentation/hybrid-cloud/platform-deployment-options.md:70]: Oracle does not seem to set allowVolumeExpansion to true either by default: https://docs.oracle.com/en-us/iaas/Content/ContEng/Tasks/contengcreatingpersistentvolumeclaim_topic-Provisioning_PVCs_on_BV.htm#:~:text=The%20cluster's%20default%20storage%20class%20is%20initially%20set%20according%20to,initially%20set%20as%20the%20default.

You also have to install the snapshot controller and snapshot class manually: https://docs.oracle.com/en-us/iaas/Content/ContEng/Tasks/contengcreatingpersistentvolumeclaim_topic-Provisioning_PVCs_on_BV.htm#contengcreatingpersistentvolumeclaim_topic-Provisioning_PVCs_on_BV-PV_From_Snapshot_CSI__section_volume-snapshot-prerequisites

**Pull request merged**